.github/workflows/continous-delivery.yml

```diff
@@ -211,15 +211,18 @@ jobs:
       - name: Create instance template for ${{ matrix.network }}
         run: |
           gcloud compute instance-templates create-with-container zebrad-${{ needs.versioning.outputs.major_version || env.GITHUB_REF_SLUG_URL }}-${{ env.GITHUB_SHA_SHORT }}-${NETWORK} \
+          --boot-disk-size 300GB \
           --boot-disk-type=pd-ssd \
           --image-project=cos-cloud \
           --image-family=cos-stable \
           --user-output-enabled \
-          --metadata google-logging-enabled=true,google-logging-use-fluentbit=true \
+          --metadata google-logging-enabled=true,google-logging-use-fluentbit=true,google-monitoring-enabled=true \
+          --container-stdin \
+          --container-tty \
           --container-image ${{ vars.GAR_BASE }}/zebrad@${{ needs.build.outputs.image_digest }} \
-          --container-env "NETWORK=${{ matrix.network }},LOG_FILE=${{ vars.CD_LOG_FILE }},SENTRY_DSN=${{ vars.SENTRY_DSN }},SHORT_SHA=${{ env.GITHUB_SHA_SHORT }}" \
-          --create-disk=name=zebrad-cache-${{ env.GITHUB_SHA_SHORT }}-${NETWORK},auto-delete=yes,size=300GB,type=pd-ssd \
-          --container-mount-disk=mount-path="/zebrad-cache",name=zebrad-cache-${{ env.GITHUB_SHA_SHORT }}-${NETWORK} \
+          --container-env "NETWORK=${{ matrix.network }},ZEBRA_CACHED_STATE_DIR=/var/cache/zebrad-cache,LOG_FILE=${{ vars.CD_LOG_FILE }},SENTRY_DSN=${{ vars.SENTRY_DSN }},SHORT_SHA=${{ env.GITHUB_SHA_SHORT }}" \
```

Comment:
This will need changes in `runtime-entrypoint.sh`. You should just be able to substitute the variable in directly, like this:
https://github.com/ZcashFoundation/zebra/blob/286ebdbf8c9e40891164821852567bc86de30012/docker/runtime-entrypoint.sh#L39

It would also be nice to set this path as the default in the `Dockerfile`, to avoid confusion.

If you're looking for an example of how runtime config creation works, see PR #6890. (But terminal colour is a bit more complicated because the variable doesn't match the config setting.)

Reply:
Solved with: [fix(docker): add the ZEBRA_CACHED_STATE_DIR as a default ENV](https://github.com/ZcashFoundation/zebra/pull/6893/commits/10c3fc9cc6e6a813f20a8f9074b108b53b9d5936)